src/agent/src/rpc.rs

```diff
@@ -1506,6 +1498,19 @@ impl agent_ttrpc::AgentService for AgentService {
 
         Ok(Empty::new())
     }
+
+    async fn set_initdata(
+        &self,
+        ctx: &TtrpcContext,
+        req: protocols::agent::SetInitdataRequest,
+    ) -> ttrpc::Result<Empty> {
+        trace_rpc_call!(ctx, "set_initdata", req);
+        crate::initdata::do_set_initdata(&req).await.map_err(|e| {
```

Comment:
Looks it will overwrite the initdata set by process-user-data in PeerPod implemented in https://github.com/confidential-containers/cloud-api-adaptor/issues/1895, Is there a plan to support initdata's hostpath mount in longer term plan?